server/routes/ratings.js

```diff
@@ -27,9 +27,47 @@ ratingsRouter.get('/getReviews', async (req, res) => {
 
 ratingsRouter.post('/updateHelpfulness', async (req, res) => {
   // console.log("receive review ID:", req.body.reviewId);
-  const Id = req.body.Id;
-  let totalReviews = await RatingApi.updateHelpfulness(Id);
+  let productId = req.body.Id;
+  let totalReviews = await RatingApi.updateHelpfulness(productId);
   res.status(204).end();
 });
 
+ratingsRouter.get('/ratingOverview', async (req, res) => {
+  let productId = req.query.Id;
+  let ratingOverview = await RatingApi.ratingOverview(productId);
+  let ratingAverage = await averageRate(ratingOverview.ratings, ratingOverview.recommended);
+  ratingOverview.ratings.average = ratingAverage[0];
+  ratingOverview.recommended = ratingAverage[1];
+  ratingOverview.ratings['1'] = ratingAverage[2];
```

Comment:
may be use for loop here instead of 6 lines?

Reply:
a loop should work, but not sure if loop is linear operation may achieve worse performance than directly assign by index or they are about the same ?